Makefile.am

```diff
@@ -90,6 +90,7 @@ AM_CPPFLAGS = \
 	-DPACKAGE_BIN_DIR=\""$(bindir)"\" \
 	-DPACKAGE_LOCALSTATE_DIR=\""$(localstatedir)"\" \
 	-DPACKAGE_LIB_DIR=\""$(libdir)"\" \
+	-DCOCKPIT_CONFIG_FILE=\""$(sysconfdir)/cockpit/cockpit.conf"\" \
```

Comment:
Could we just derive this from PACKAGE_SYSCONF_DIR instead, and drop this constant?